src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/Blocks/import_panel.html.twig

```diff
@@ -28,152 +28,149 @@
+      </ul>
+    </div>
+
+    <hr>
+
+    <div class="form-group hidden-xs-up">
+      {{ form_errors(importForm.csv) }}
+      {{ form_widget(importForm.csv, {'attr': {'class': 'js-import-file-input'}}) }}
+    </div>
+
+    <div class="form-group js-file-upload-form-group">
+      <label class="form-control-label">{{ 'Select a file to import'|trans }}</label>
+      {{ form_errors(importForm.file) }}
+      {{ form_widget(importForm.file, {'attr': {'class': 'js-import-file', 'data-max-file-upload-size': maxFileUploadSize }}) }}
+      <span>{{ 'or'|trans({}, 'Admin.Global') }}</span>
+      <button type="button"
+              class="btn btn-outline-primary btn-sm js-from-files-history-btn"
+              {% if importFileNames is empty %}disabled{% endif %}
+      >
+        <span class="badge badge-primary js-files-history-number">{{ importFileNames|length }}</span>
+        {{ 'Choose from history / FTP'|trans({}, 'Admin.Advparameters.Feature') }}
+      </button>
+      <small class="form-text">{{ 'Allowed formats: .csv, .xls, .xlsx, .xlst, .ods, .ots'|trans({}, 'Admin.Advparameters.Help') }}</small>
+      <small class="form-text">{{ 'Only UTF-8 and ISO 8859-1 encodings are allowed'|trans({}, 'Admin.Advparameters.Help') }}</small>
+      <small class="form-text">{{ 'You can also upload your file via FTP to the following directory: %s .'|trans({'%s': importDirectory}, 'Admin.Advparameters.Help') }}</small>
+    </div>
+
+    <div class="alert alert-danger d-none js-import-file-error" role="alert">
+      <p class="alert-text">
+        <strong class="js-file-data"></strong>: <span class="js-error-message"></span>
+      </p>
+    </div>
+
+    <div class="alert alert-success d-none js-import-file-alert" role="alert">
+      <button type="button"
+              class="close btn btn-outline-secondary js-change-import-file-btn"
+              aria-label="{{ 'Change'|trans({}, 'Admin.Actions') }}"
+      >
+        <span aria-hidden="true"><i class="material-icons">edit</i></span>
+      </button>
+      <p class="alert-text js-import-file"></p>
+    </div>
+
+    {% block import_file_history_block %}
+      {% include '@PrestaShop/Admin/Configure/AdvancedParameters/Blocks/import_file_history.html.twig' %}
+    {% endblock %}
+
+    <hr>
+
+    <div class="form-group">
+      {{ ps.label_with_help(('Language of the file'|trans), ('The locale must be installed'|trans({}, 'Admin.Advparameters.Notification'))) }}
+      {{ form_errors(importForm.iso_lang) }}
+      {{ form_widget(importForm.iso_lang) }}
+    </div>
+
+    <div class="form-group">
+      <label class="form-control-label">{{ 'Field separator'|trans }}</label>
+      {{ form_errors(importForm.separator) }}
+      {{ form_widget(importForm.separator) }}
+      <small class="form-text">{{ 'e.g. '|trans({}, 'Admin.Advparameters.Help') }} 1; Blouse; 129.90; 5</small>
```

Comment:
There is a blank space at the end of the string.

Reply:
translation key contains this space. :)